apollo-router/src/batching.rs

```diff
@@ -478,7 +474,9 @@ mod tests {
             .unzip();
 
         // Assemble them
-        let (op_name, _context, request, txs) = assemble_batch(requests).await;
+        let (op_name, _context, request, txs) = assemble_batch(requests)
+            .await
+            .expect("it can assemble a batch");
```

Comment:
If we expect this to never fail, should `assemble_batch` not return a result?

Reply:
Not sure I understand. This is a test, so if it does fail, the test will fail and we have an issue to resolve.